docs/GOVERNANCE.md

```diff
@@ -51,10 +51,12 @@ A contributor to the project must express interest in becoming a maintainer.
 The CB has the authority to add or remove maintainers.
 
 ## Changes in governance
-The CB supervises changes in governance.  In the case where the CB cannot or elects not 
-to do so, a quorum of the maintainers select the new CB as follows:
 
-The consensus builder may be appointed for a fixed term or it may be a lifetime appointment.  To initiate a change of consensus builder, or a change in the length of the appointment,  a GitHub PR must be opened.  If a fixed term is specified, the PR should be opened no earlier than 6 weeks before the end of the CB's term, and  should be kept open for no less than 4 weeks. The PR can only be merged after the end of the last CB's term, with more +1 than -1 in the binding votes.
+The CB supervises changes in governance, but a majority of maintainers must vote +1 on the PR.  
+
+## Changes in the consensus builder
+
+The consensus builder may be appointed for a fixed term or it may be a lifetime appointment.  To initiate a change of consensus builder, or a change in the length of the appointment,  a GitHub PR must be opened.  If a fixed term is specified, the PR should be opened no earlier than 6 weeks before the end of the CB's term. If there is not a fixed term appointment the PR may be opened at any time.  In either case the PR must be kept open for no less than 4 weeks.  Additionally the PR can only be merged with more +1 than -1 in the binding votes.
```

Comment:
I suggest to add three commas, one after *"If there is not a fixed term appointment"*,  one after *"In either case"*, and one after  *"Additionally"* (not sure if they are mandatory), i.e.:

```suggestion
The consensus builder may be appointed for a fixed term or it may be a lifetime appointment.  To initiate a change of consensus builder, or a change in the length of the appointment,  a GitHub PR must be opened.  If a fixed term is specified, the PR should be opened no earlier than 6 weeks before the end of the CB's term. If there is not a fixed term appointment, the PR may be opened at any time.  In either case, the PR must be kept open for no less than 4 weeks.  Additionally, the PR can only be merged with more +1 than -1 in the binding votes.
```